lib/backend/etcdv3/etcdv3.go

```diff
@@ -395,7 +395,7 @@ func (c *etcdV3Client) EnsureInitialized() error {
 func (c *etcdV3Client) Clean() error {
 	log.Warning("Cleaning etcdv3 datastore of all Calico data")
 	_, err := c.etcdClient.Txn(context.Background()).If().Then(
-		clientv3.OpDelete("/calico", clientv3.WithPrefix()),
+		clientv3.OpDelete("/calico/", clientv3.WithPrefix()),
```

Comment:
Why this change? It's prefix operation so should be fine, right?

Reply:
it's strictly more correct since this distinguishes from /calicov2/blah for example.